_layouts/contribution_guide.html

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
@kmadof  zrób zamiast tego layout page i użyj jako bazy layout post

Reply:
Coś jest nie tak kieyd wezmę w page.html po prorstu

```yaml
---
layout: post
---
```

Muszę jeszcze to poprawić

Reply:
Ponadto jak weźmiemy `post` to dostajemy też datę na stronie. Wygląda to tak. Zajmę się tym:
![image](https://user-images.githubusercontent.com/1680172/103357930-a81d6700-4ab4-11eb-8934-e11812c896a5.png)

Reply:
Tak, tak, trzeba pokasować rzeczy :) tak samo z dołu.

Reply:
Poszło.